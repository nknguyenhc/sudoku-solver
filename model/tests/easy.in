_ 1 _ 3 _ _ _ _ 2
9 _ 8 5 1 _ 3 7 _
7 6 _ _ _ _ _ 8 5
_ 5 _ _ 2 _ 4 9 6
_ _ _ 7 5 4 _ _ _
2 8 _ 1 6 _ _ 5 _
6 7 2 _ 4 _ _ 3 _
8 _ _ _ _ 5 _ 4 9
5 _ _ _ _ _ 8 _ 7