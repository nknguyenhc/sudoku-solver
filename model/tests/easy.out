4 1 5 3 8 7 9 6 2
9 2 8 5 1 6 3 7 4
7 6 3 4 9 2 1 8 5
1 5 7 8 2 3 4 9 6
3 9 6 7 5 4 2 1 8
2 8 4 1 6 9 7 5 3
6 7 2 9 4 8 5 3 1
8 3 1 2 7 5 6 4 9
5 4 9 6 3 1 8 2 7